src/components/victory-chart.jsx

```diff
@@ -1,8 +1,7 @@
 import React from "react";
 import Radium from "radium";
 import _ from "lodash";
-import log from "../log";
-import Util from "../util";
+import Util from "victory-util";
```

Comment:
You could just `import { Collection, Log } from 'victory-util';` and avoid referencing `Util` below.